crates/eframe/src/web/web_painter_glow.rs

```diff
@@ -43,8 +38,8 @@ impl WebPainter for WebPainterGlow {
         self.painter.max_texture_side()
     }
 
-    fn canvas_id(&self) -> &str {
-        &self.canvas_id
+    fn canvas(&self) -> HtmlCanvasElement {
+        self.canvas.clone()
```

Comment:
This could return a reference instead, then all `&runner.canvas()` everywhere could just be `runner.canvas()`